src/Microsoft.Diagnostics.ExtensionCommands/Output/Dml.cs

```diff
@@ -0,0 +1,306 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.Diagnostics.Runtime;
+
+namespace Microsoft.Diagnostics.ExtensionCommands.Output
+{
+    internal static class Dml
+    {
+        private static DmlDumpObject s_dumpObj;
+        private static DmlDumpHeap s_dumpHeap;
+        private static DmlBold s_bold;
+        private static DmlListNearObj s_listNearObj;
+        private static DmlDumpDomain s_dumpDomain;
+        private static DmlThread s_thread;
+
+        /// <summary>
+        /// Runs !dumpobj on the given pointer or ClrObject.  If a ClrObject is invalid,
+        /// this will instead link to !verifyobj.
+        /// </summary>
+        public static DmlFormat DumpObj => s_dumpObj ??= new();
```

Comment:
Nit: By just glancing at this line using new() here is confusing because it isn't obvious what overload of DmlFormat is being set.

Reply:
IIRC the compiler warnings force us to use new(), but maybe it allows an explicit new in this case.  I'll check.

Reply:
Don't go overboard on fixing this. It's fine the way it is.